### PR TITLE
Fix Upgrade Scenarios fails

### PR DESCRIPTION
**Pull request by jyejare**
Fixes Upgrade Scenarios:

1. Capsule Sync
2. Client package installation

depends on https://github.com/SatelliteQE/nailgun/pull/494

### CODE REVIEW

File: upgrade_tests/test_scenarios/test_capsule.py [hunk 1]
```diff
@@ -43,9 +43,9 @@ class Scenario_capsule_sync(TestCase):
     """
     cls_name = 'Scenario_capsule_sync'
     sat_host = get_satellite_host()
```
Comment (san7ket) [inline upgrade_tests/test_scenarios/test_capsule.py]: Can we have this in init ?

File: upgrade_tests/test_scenarios/test_capsule.py [hunk 2]
```diff
@@ -43,9 +43,9 @@ class Scenario_capsule_sync(TestCase):
...
+    env.host_string = sat_host
```
Comment (san7ket) [inline upgrade_tests/test_scenarios/test_capsule.py]: Same as above, it is needed for most modules

File: upgrade_tests/test_scenarios/test_capsule.py (hunk 1 above)
Comment (jyejare) [reply upgrade_tests/test_scenarios/test_capsule.py]: Not required as its declared at a class scope and will be initialized when the class is called, No need of __init__.

File: upgrade_tests/test_scenarios/test_capsule.py (hunk 2 above)
Comment (jyejare) [reply upgrade_tests/test_scenarios/test_capsule.py]: Same as above answer

**Pull request merged**